File: java/src/main/java/com/languageweaver/sdk/samples/languagedetection/file/CloudFileLanguageDetectionService.java
package com.languageweaver.sdk.samples.languagedetection.file;

import com.languageweaver.sdk.common.cloud.CloudLanguageWeaverClient;
import com.languageweaver.sdk.common.constants.Format;
import com.languageweaver.sdk.languagedetection.cloud.request.CloudFileLanguageDetectionRequest;
import com.languageweaver.sdk.languagedetection.cloud.result.CloudLanguageDetectionResult;

import java.nio.file.Paths;

public class CloudFileLanguageDetectionService {
    public static void main(String[] args) throws Exception {
        try (CloudLanguageWeaverClient lwClient = new CloudLanguageWeaverClient().build()) {
            CloudFileLanguageDetectionRequest cloudFileLanguageDetectionRequest = new CloudFileLanguageDetectionRequest()
                    .setInputFile(Paths.get("java", "src", "main", "resources", "input", "input1.txt").toFile().getAbsolutePath())
                    .setInputFormat(Format.PLAIN);

            final CloudLanguageDetectionResult cloudLanguageDetectionResult = lwClient.detectLanguage(cloudFileLanguageDetectionRequest);

            if (cloudLanguageDetectionResult.getLanguages() != null) {
                cloudLanguageDetectionResult.getLanguages()
                        .forEach(language -> System.out.println(language.getName()));
            }
        }
    }
}

File: dotnet/Samples/LanguageDetection/File/CloudFileLanguageDetectionService.cs
using LanguageWeaver.Sdk.Common.Cloud;
using LanguageWeaver.Sdk.Constants;
using LanguageWeaver.Sdk.LanguageDetection.Cloud.Request;

namespace LanguageWeaver.Sdk.Samples.LanguageDetection.File;

public class CloudFileLanguageDetectionService
{
    public static void Main()
    {
        using var lwClient = new CloudLanguageWeaverClient().Build();
        var cloudFileLanguageDetectionRequest = new CloudFileLanguageDetectionRequest()
        {
            InputFile = Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName,
                "Resources", "Input", "input1.txt"),
            InputFormat = Format.Plain
        };

        var cloudLanguageDetectionResult = lwClient.DetectLanguage(cloudFileLanguageDetectionRequest);

        cloudLanguageDetectionResult.Languages
            .ForEach(language => Console.WriteLine(language.Name));
    }
}